src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -888,6 +900,9 @@
   <data name="ERR_FixedMustInit" xml:space="preserve">
     <value>You must provide an initializer in a fixed or using statement declaration</value>
   </data>
+  <data name="ERR_You_must_provide_an_initializer_for_a_readonly_variable" xml:space="preserve">
+    <value>You must provide an initializer for a 'readonly' variable</value>
```

Comment:
I don't think we should be getting personal in error messages. Perhaps should say something like this instead: "Declaration of a readonly variable must have an initializer."